feature/aks-cluster-creation/README.md

```diff
@@ -13,6 +13,7 @@ The Azure Kubernetes Service (AKS) is the service for deploying, managing, and s
 - Different network drivers (basic/advance)
 - Different resource groups
 - Possibility to create new network or use existing network and resource group. AKS will just join to the existing network.
+- Cluster autoscaling
```

Comment:
what do you think about moving all terraform file to subdirectory of `design-doc/aks`? That would feel natural for me to later merge it to develop and have all those PoC scripts in there.

Reply:
Nice idea, it's good to have "somewhere" the terraform scripts, even after merged the branch into develop.

Reply:
so would you delete this directory?